pallets/dkg-proposal-handler/Cargo.toml

```diff
@@ -15,7 +15,7 @@ targets = ["x86_64-unknown-linux-gnu"]
 codec = { package = "parity-scale-codec", version = "2.0.0", features = ["derive"], default-features = false }
 scale-info = { version = "1.0", default-features = false, features = ["derive"] }
 
-
+pallet-dkg-proposals = { path = "../dkg-proposals", default-features = false }
```

Comment:
I don't believe this should be needed in the main dependencies. If anything, it should be an optional dependency `optional = true` and you should be able to remove it from `dev-dependencies` as well.

Reply:
removed that dependency and runtime-benchmark features that were used for benchmarking.